documentation/concepts.md

```diff
@@ -48,13 +77,20 @@ cluster [controller](#controller). The leader sequences each message in the
 partition and sends back an acknowledgement to publishers upon committing a
 message to the log. A message is committed to the log once it has been
 replicated to the partition's
-[in-sync replica set (ISR)](#in-sync-replica-set-isr).
+[in-sync replica set (ISR)](#in-sync-replica-set-isr). [@Tyler: what happens when 
+replication fails? Do we lose the event?] 
```

Comment:
Please add what you just said to the text. It is vital info, since I will not be the only one asking this question. The proposal for text could be "Is it possible we can loose events because of this?  The answer is - it depends on how things are configured. If the message is published with the LEADER ack policy (default), an ack is sent back to the client as soon as the leader has stored the message. If the message is published with the ALL ack policy, the ack is sent only after all members of the ISR have stored it. Thus, if the ISR is unavailable, no ack will be sent indicating there's no guarantee the message was committed. A minimum ISR size can also be configured to provide a high level of durability, but this creates a trade-off with availability (see docs here)."